### ISSUE TITLE
taxon image display bug

### ISSUE THREAD

**Issue by touretzkyds** [touretzkyds/KnowledgeGraphDemo#43]
In the latest version of demo2d, display the properties of some taxon with a taxon image. When you then hover the mouse over the image, you get an error. This error appears to be caused by event.cyTarget being undefined, so the variable "node" is undefined, so the reference node.qtip raises an error.

![callicebus](https://user-images.githubusercontent.com/1978092/175804394-e89f9779-8413-414d-afaa-5e044695f310.png)

**Comment by deyaliao** [comment]
I found a new way to create the tooltip effect when hovering over an image, so this is no longer a bug.

**Comment by touretzkyds** [comment]
Yes, hovering is no longer a problem. But some taxon images are failing to load. Go to "Iguana" and you'll see.

**Comment by deyaliao** [comment]
There was a bug in the way I sourced my images. I think it should be fixed now.

**Comment by touretzkyds** [comment]
I'm still not seeing an image for Iguana.
